### PR TITLE
🐛Source Instagram: Capture media_product_type column by adding it to schema

### LINKED ISSUE #28260: Source Instagram:  It does not capture media_product_type column #28180
### What area the feature impact?

Connectors

### Revelant Information

Connector Name
source-instagram

Connector Version
1.0.9

What step the error happened?
Configuring a new connector

Revelant information
The Instagram connector is currently not capturing the media_product_type field because of this it is not possible to distinguish the media type. Therefore, it is necessary to add this field to capture this information.

Current output
image

Currently, the Airbyte connector is searching for the parameter "media_product_type," but the correct parameter to use is "media_type," as shown in the official Facebook documentation.

Expected output
image

https://developers.facebook.com/docs/instagram-api/reference/ig-user/media

Output airbyte on aws athena, using o dbeaver to connect.

media_product_type: Surface where the media is published. Can be AD, FEED, STORY or REELS.

### PR DESCRIPTION
**Pull request by pabloescoder**
## What
Resolves #28260 

## How
Added the missing media_product_type property in the schema files `media.json` and `stories.json`

## Recommended reading order
1. `media.json`
2. `stories.json`

## 🚨 User Impact 🚨
No breaking changes.

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: docs/integrations/sources/instagram.md [hunk 1]
```diff
@@ -86,7 +86,8 @@ AirbyteRecords are required to conform to the [Airbyte type](https://docs.airbyt
 ## Changelog
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                   |
-| :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+|:--------|:-----------|:---------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| 1.0.12  | 2023-10-30 | [31947](https://github.com/airbytehq/airbyte/pull/31947) | Capture media_product_type column                                                                                         |
```
Comment (marcosmarxm) [inline docs/integrations/sources/instagram.md:90]: Be more specific `Capture media_product_type column in media and stories stream`
Please bump the metadata and dockerfile version too.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: docs/integrations/sources/instagram.md (hunk 1 above)
Comment (pabloescoder) [reply docs/integrations/sources/instagram.md]: @marcosmarxm Made the requested changes!

**Review by marcosmarxm — APPROVED** [review]
Thanks @pabloescoder change looks correct. At the moment integration are failing and we're going to work on them during this week or next one.

**Comment by pabloescoder** [comment]
> Thanks @pabloescoder change looks correct. At the moment integration are failing and we're going to work on them during this week or next one.

Sounds great. Thank you! @marcosmarxm

**Comment by marcosmarxm** [comment]
This contribution code was merged in https://github.com/airbytehq/airbyte/pull/32423 thanks @pabloescoder

**Pull request closed without merging**